scripts/publish_metadata.py

```diff
@@ -0,0 +1,27 @@
+from multibase import multibase
+from multicodec import multicodec
+from aea.helpers.cid import to_v1
+from aea_cli_ipfs.ipfs_utils import IPFSTool
+from scripts.generate_metadata import METADATA_FILE_PATH
+
+PREFIX = "f01701220"
+IPFS_PREFIX_LENGTH = 6
+
+
+def push_metadata_to_ipfs() -> None:
+    response = IPFSTool().client.add(
+        METADATA_FILE_PATH, pin=True, recursive=True, wrap_with_directory=False
+    )
+    cid_bytes = multibase.decode(to_v1(response["Hash"]))
```

Comment:
Let's extract a constant for the expected hash key, handle the key error, and print some useful message.